src/ordered_dict.jl

```diff
@@ -428,12 +424,6 @@ function start(t::OrderedDict)
 end
 done(t::OrderedDict, i::Int) = i > length(t.keys)
 next(t::OrderedDict, i::Int) = (Pair(t.keys[i],t.vals[i]), i+1)
-
-if isdefined(Base, :KeySet) # 0.7.0-DEV.2722
-    next(v::Base.KeySet{K,T}, i) where {K,T<:OrderedDict{K}} = (v.dict.keys[i], i+1)
-else
-    next(v::Base.KeyIterator{T}, i) where {T<:OrderedDict} = (v.dict.keys[i], i+1)
-end
```

Comment:
Again, I am not sure you intended to delete both branches of this `if` and `else`